examples/redis_sync/plan.fmf

```diff
@@ -0,0 +1,38 @@
+summary: Plan 1
+description: |
+  This is example how to use redis for data synchronization between guests
+  how to sednd data to server:
+      redis-cli -h $GUEST set variable value
+  how to get data from server:
+      redis-cli -h $GUEST get variable
+  If you need more features like waiting for events or atomic commands or deleting keys
+  see doc for redis commands https://redis.io/commands/
+provision:
+  - how: minute
```

Comment:
Can we use `virtual` instead to make the example working outside of the internal network?

Reply:
theoretically yes, problem could be when virt machines are not in same network so unable to see each other, but in this case could be okay. I expect that `virt` runner will run both machines in one network to be accessible @psss

Reply:
Yes, the `virtual` provision should work fine for the multihost scenarios when `connection: system` is used.